File: Jotter/linked lists.py
class LL:
    def __init__(self):
        self.head = None

class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

listhead = LL()

node1 = Node("a")
node2 = Node("b")
node3 = Node("hello world!")

listhead.head = node1
node1.next = node2
node2.next = node3

print((node1.next).data)

while list1 and list2:
    if list1.val <= list2.val:
        tail.next = list1
        list1 = list1.next
    else:
        tail.next = list2
        list2 = list2.next
    tail = tail.next
if list1:
    tail.next = list1
elif list2:
    tail.next = list2
return dummy.next

File: Jotter/linked_lists.js
var head;

class Node{

    constructor(val){
        this.data = val
        this.next = null
    }
}

function print(){
    n = head
    while(n != null){ 
        console.log(n.data)
        n = n.next  
    }
}

function push(vall){
    var new_node = new Node(vall)

    new_node.next = head
    head = new_node
}

head = new Node(1)
sec = new Node(2)
head.next = sec

print()
push(0)
print()